src/shared/featuredCategories.js

```diff
@@ -1,22 +1,23 @@
+import eggs from "../assets/featuredCategories/egg.jpg";
+import toast from "../assets/featuredCategories/toast.jpg";
+import breakfast from "../assets/featuredCategories/breakfast.jpg";
+import drink from "../assets/featuredCategories/drink.jpg";
+
 export const featuredCategories = [
   {
-    picturePath:
-      "https://plus.unsplash.com/premium_photo-1676234917179-a7b1ca98c984?w=700&auto=format&fit=crop&q=60&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8NDF8fGZvb2R8ZW58MHx8MHx8fDA%3D",
+    picturePath: eggs,
```

Comment:
I think it makes more sense to use the path `../assets/featuredCategories/egg.jpg`